### PR TITLE
provide Netlify and GitHub Pages as unbiased choices

### PR DESCRIPTION
**Pull request by jelhan**
The current text ~states~ could be easily misinterpreted that most addons in the org uses Netlify. This wasn't matching my experience and therefore I quickly counted. 3 addons are currently using Netlify and 7 addons are using GitHub Pages. So it's more the other way around.

To be honest I'm not an expert on the topic and haven't used Netlify myself yet. There might be arguments to recommend it over GitHub Pages. But the current usage by addons in this org is more an argument for the opposite.

To avoid a opinionated discussion about the topic I propose that we present both as valid choices for now.

---

To make my numbers comprehensible here are the addons that I found grouped by their hosting. I have only considered addons that had an URL configured on GitHub.

These addons are using Netlify:

- https://github.com/adopted-ember-addons/ember-impagination
- https://github.com/adopted-ember-addons/ember-stripe-elements
- https://github.com/adopted-ember-addons/emberx-select

These addons are using GitHub Pages:

- https://github.com/adopted-ember-addons/ember-autoresize
- https://github.com/adopted-ember-addons/ember-keyboard
- https://github.com/adopted-ember-addons/ember-file-upload
- https://github.com/adopted-ember-addons/ember-sortable
- https://github.com/adopted-ember-addons/ember-page-title
- https://github.com/adopted-ember-addons/ember-collapsible-panel
- https://github.com/adopted-ember-addons/ember-electron

/cc @Alonski

### CODE REVIEW

**Comment by MelSumner** [comment]
@jelhan I think we meant to indicate that we mostly use Netlify across the Ember org as a whole, which is accurate. Netlify has been really great to open source, Ember in particular, and we are very grateful to them for their incredible support. 

That being said, I don't mind it if we also include information about other ways to deploy sites.

**Review by knownasilya — APPROVED** [review]
Looks good to me.

**Comment by jelhan** [comment]
> I think we meant to indicate that we mostly use Netlify across the Ember org as a whole

This highlights that the term _org_ might be a little bit vague in the document. Does it refer to ember organization or to adopted-ember-addons? I think it would be helpful to clarify. I always only thought about it in the latter way. This is not an official project, is it?

**Comment by jelhan** [comment]
Anything left before this one could be merged? More than one approval required? I don't have merge permissions myself. Just in case you expected me to hit the button. :wink:

**Comment by knownasilya** [comment]
Thanks @jelhan

**Pull request merged**
